lms/views/lti/oidc.py

```diff
@@ -0,0 +1,52 @@
+import uuid
+from urllib.parse import urlencode
+
+from pyramid.httpexceptions import HTTPForbidden, HTTPFound
+from pyramid.view import view_config
+
+from lms.services import LTIRegistrationService
+from lms.validation import OIDCRequestSchema
+
+
+@view_config(
+    route_name="lti.oidc",
+    request_method=("POST", "GET"),
+    schema=OIDCRequestSchema,
+)
+def oidc_view(request):
```

Comment:
Not much going on here

- Check we have a registration
- Redirect to `registration.auth_login_url`

It looks a bit longer with the dictionary of values here but I don't think we should hide it away. All the values are very OIDC view specific and won't be used elsewhere.